docs/password_auth_providers.rst

```diff
@@ -37,3 +37,15 @@ Password auth provider classes must provide the following methods:
 
     The method should return a Twisted ``Deferred`` object, which resolves to
     ``True`` if authentication is successful, and ``False`` if not.
+
+Optional methods
+----------------
+
+Password provider classes may optionally provide the following methods.
+
+*class* ``SomeProvider.get_db_schema_files()``
+
+    This method, if implemented, should return an Iterable of ``(name,
```

Comment:
Is it worth clarifying that this must be a file name?

Reply:
It doesn't really need to be a filename - any unique identifier will do